Views/js/mds_collivery.js

```diff
@@ -103,6 +103,39 @@ jQuery(document).ready(function () {
         }
     });
 
+    //Used to update order with an International Waybill
+    jQuery('#update_order').click(function(event){
+        if(jQuery('#waybill_number').val().trim().length > 0){
+            event.preventDefault();
+            var datastring = jQuery("#intl_waybill_link").serialize();
+            jQuery.ajax({
+                type: "POST",
+                url: ajaxurl,
+                data: 'action=update_international_order_admin&' + datastring,
+                success: function (data) {
+                    jQuery("#api_results").html('<div style="font-size: 15px;margin:15px 0 0 39px;color:black;">' + data.message + '</div>');
+                    console.log('Success -- returning data');
```

Comment:
Please remove the `console` statements before merging

Reply:
I've removed all informative test console logs from the solutions.